.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build
 
 on:
   pull_request:
-    branches: [main]
+    branches: [main, console-mvp]
```

Comment:
I am seeing some strange behavior from earlier versions of nodejs with the web sdk. Removing them for now to unblock us. Will have to come back to this when we merge it into main